tests/users/test_api_resend_email.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+from flask import json
+
+from app.database.models.user import UserModel
+from app.database.sqlalchemy_extension import db
+from tests.base_test_case import BaseTestCase
+from tests.test_data import user1
+
+
+class TestResendEmailApi(BaseTestCase):
+
+    def setUp(self):
+        super(TestResendEmailApi, self).setUp()
+
+        self.verified_user = UserModel(
+            name=user1["name"] + "    Example",
```

Comment:
I don't understand the need for spaces here

Reply:
@satya7289 can you remove those spaces if there is not a reason for it?

Reply:
ok